### PR TITLE
Removed code for updating menu.json file for desktop

### PR DESCRIPTION
**Pull request by amityevaleCel**

### CODE REVIEW

**Comment by amityevaleCel** [comment]
Change-log: 
1. Application will not try to update Menu.json file for desktop as per defect description.
2. For respective desktop/phone profile displaying message only for that profile only
3. Showing message to update file with file path with the menu item to be updated by developer

File: packages/ext-build/generate/viewpackage.js [hunk 1]
```diff
@@ -100,7 +100,20 @@ try {
 				menuJson.children.push(JSON.parse(newMenu))
 				fs.writeFileSync(menuFile, JSON.stringify(menuJson), 'utf8')
 			}
-			console.log(help.menuText(menuFile, item, itemphone));
+			console.log(help.menuText(menuFile, item, itemphone)); */
+			var item,
+				menuFile;
+
+			if (profile === 'desktop') {
```
Comment (jnsencha) [inline packages/ext-build/generate/viewpackage.js]: @amityevaleCel what are the implications here for profiles that are not defaults, e.g. a customer defined profile.

Comment (jnsencha) [reply packages/ext-build/generate/viewpackage.js]: @wemersonjanuario I think we really need this to be more dynamic don't we? Ext JS normal configs in app.json allow for any number of profiles that the developer can name whatever they wish.

Comment (wemersonjanuario) [reply packages/ext-build/generate/viewpackage.js]: My inputs:
@jnsencha @vajrakumar 
We do not want to create/change unnecessary files anymore (menu, MenuView) and do not want to provide commands that feeds templates/or hello world apps. 

Command should not try to create a menu option by itself  for any profile
Command should not output that menu.json was updated. Customer maybe does not have this file.

Command should only do what it intends to do.

- create a view package for the specified profiles, if no profile specified , view package files should be stored in shared dir or /app/src if its a Cmd based upgraded app
- Output message "View package "{name}" was creates at "dir"

Other option is to ask user about creating menu entry or no, or hide this behavior and if you would like to create menu entries user should pass an option like --add-to-menu

I would go to first option (removing menu updates/option for all progfiles ) and just create view package and output message that package X was create at Y Dir. Will be easier, cleaner and non-buggy


One bonus for this, would be creating a View.scss file as well or an option --include-scss ;)

Comment (amityevaleCel) [reply packages/ext-build/generate/viewpackage.js]: As per comments updated below changes:

  1. Added Parameterised support for ext-build generate viewpackage command while maintaining backward functionality (index based support). New parameter names will be profile (-p for short) & viewname (-v for short)
  2. If 'profile' option is not included in command then view will be created in shared directory
  3. Added parameterised commands in help documentation which is displayed when we type 'ext-build' command in console.
  4. Now SCSS file for a viewpackage will only be generated if --includes-scss OR -i option is added while generating view package.
  5. For migrated Sencha cmd app view will be created at shared directory appRootDirectory/app/view

**Review by jnsencha — APPROVED** [review]

**Pull request merged**